app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java

```diff
@@ -130,10 +130,7 @@ protected void onStart() {
 
         mGoogleApiClient.registerConnectionCallbacks(this);
         mGoogleApiClient.registerConnectionFailedListener(this);
-
-        if (PrefUtils.isSignedIn(this)) {
-            mGoogleApiClient.connect();
-        }
+        mGoogleApiClient.connect();
```

Comment:
I think we should implement "proper" logout and automatic sign-in. 

With automatic sign-in, we just need to update SharedPrefs and we should remove the login page from `FirstStartActivity` if the user automatically logins.